site/content/docs/_index.md

```diff
@@ -0,0 +1,172 @@
++++
+title = "Docs"
+layout = "single"
++++
+
+## Overview
+
+WebAssembly is a way to safely run code compiled in other languages.
+Runtimes execute WebAssembly Modules (Wasm), which are most often binaries with a `.wasm` extension.
+Most WebAssembly modules import functions from the host, to perform tasks that are otherwise disallowed by their sandbox.
+The most commonly imported functions are called WASI, which allow access to system resources such as the console or files.
+
+wazero is a WebAssembly runtime, written completely in Go. It has no platform dependencies, so can be used in any environment supported by Go.
+
+## API
+
+Being a Go library, which we document wazero's API via [godoc][godoc].
+
+## Terminology
+
+Wazero has consistent terminology used inside the codebase which may be new to you, or different than another WebAssembly runtime.
+This section covers the most commonly used vocabulary. Terms rarely used may also be defined inline in individual sections.
+
+* Host - Host is a WebAssembly concept that refers to the process that embeds a WebAssembly runtime. In wazero, the host is a program written in Go.
+* Binary - Binary is a WebAssembly module, compiled from source such as C, Rust or Tinygo. This is also called Wasm or a guest, and usually is a file with a `.wasm` extension. This is the code wazero runs.
+* Module - Module an instance of a Binary, which usually exports functions that can be invoked by the embedder. It can also import functions from the host to perform tasks not defined in the WebAssembly Core Specification, such as I/O.
```

Comment:
```suggestion
* [Module][Module] - Module an instance of a Binary, which usually exports functions that can be invoked by the embedder. It can also import functions from the host to perform tasks not defined in the WebAssembly Core Specification, such as I/O.
```